components/svg/HeroImage.tsx

```diff
@@ -1,11 +1,20 @@
+"use client";
+
+import { motion } from "framer-motion";
+
 export const HeroImage = () => (
   <svg
-    className="max-h-[33.25rem] w-full max-w-[49.875rem]"
+    className="max-h-[33.25rem] w-full max-w-[49.875rem] min-[1440px]:mb-[6.56rem] min-[1440px]:min-w-[49.875rem]"
```

Comment:
You really shouldn't need to use such large margins as 6.56 rem. This is why it is important to implement Flexbox correctly from the very beginning. I am happy to review this together during office hours or one-on-one. Whatever I can do to help, please let me know.